### PR TITLE
Paloalto panos show mac all

### PR DESCRIPTION
**Pull request by dainok**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
paloalto_panos_show_mac_all.template

##### SUMMARY
Implementing "show mac all" for Palo Alto firewalls used as switches.

### CODE REVIEW

**Comment by dainok** [comment]
Removed not applicable files and added the `-> Error` as requested.

File: templates/paloalto_panos_show_mac_all.template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Value INTERFACE (\S+)
+Value STATUS (\S+)
+Value TTL (\d+)
+
+Start
+  ^\s*--- -> Start_record
+
+Start_record
+  ^${VLAN}\s+${MAC}\s+${INTERFACE}\s+${STATUS}\s+${TTL} -> Record
+  ^.* -> Error
```
Comment (FragmentedPacket) [inline templates/paloalto_panos_show_mac_all.template]: Please update to:
```
  ^\s*$$
  ^. -> Error
```

Comment (dainok) [reply templates/paloalto_panos_show_mac_all.template]: fixed as requested

File: templates/index [hunk 2]
```diff
@@ -293,6 +293,7 @@ paloalto_panos_show_interface_logical.template, .*, paloalto_panos, sh[[ow]] int
 paloalto_panos_show_counter_global.template, .*, paloalto_panos, sh[[ow]] coun[[ter]] glo[[bal]]
 paloalto_panos_show_system_info.template, .*, paloalto_panos, sh[[ow]] sys[[tem]] in[[fo]]
 paloalto_panos_show_jobs_all.template, .*, paloalto_panos, sh[[ow]] jo[[bs]] all
+paloalto_panos_show_mac_all.template, .*, paloalto_panos, sh[[ow]] mac all
```
Comment (FragmentedPacket) [inline templates/index]: This is in conflict since I merged the other palo alto PR. Mind adding the show arp command on L295 above the mac command?

Comment (dainok) [reply templates/index]: fixed, thx!

**Pull request merged**